### ISSUE TITLE
Theme installation removes all module hooks

### ISSUE THREAD

**Issue by getdatakick** [thirtybees/thirtybees#1104]
Since 1.1.0, theme installation removes **all** hooks for any module mentioned in `config.xml` file, and replace them with the hook list defined in this file.

This should be done for **displayable** hooks only -- those hooks that starts with `display` (+couple of exception like `Header` hook). 

Theme should **NOT** touch other hooks, as they do not impact the way module is presented on frontend. For example `actionBeforeProductSave` or `actionCronJobs`.

Related problem is that theme `config.xml` file describes specific version of the module. But merchant can have different version installed, with different set of hooks. By installing theme on a system with *upgraded* module we are effectively crippling its (new) functionality. Example:

1. theme knows about module `revws` at version `1.2.0` and contains *all* its hooks
2. merchant purchase premium version of `revws` module `2.2.0` which has additional hooks, like `actionCronJob`.
3. merchants installs theme after he updated module. Core will now unregister `actionCronJob`. And because it's not part of the config.xml, it will not be registered again.

The result is broken module. 

The proposed fix is to ignore all non-displayable hooks during theme export / import.

Related to issue #1103

**Comment by getdatakick** [comment]
Blocked by issue #1105
